src/vs/workbench/services/untitled/common/untitledEditorService.ts

```diff
@@ -130,7 +154,7 @@ export class UntitledEditorService implements IUntitledEditorService {
 			.map((i) => i.getResource());
 	}
 
-	public createOrGet(resource?: URI, modeId?: string): UntitledEditorInput {
+	public createOrGet(resource?: URI, modeId?: string, restoreResource?: URI): UntitledEditorInput {
```

Comment:
It's changing to align with text files in https://github.com/Microsoft/vscode/issues/14067